index.html

```diff
@@ -1,179 +1,228 @@
+            margin: 0 auto;
         }
 
         .container {
+            max-width: 1020px;
+            margin: 0 auto;
+            padding: 0 15px;
             display: flex;
-            max-width: 1050px;
             flex-direction: column;
             align-items: center;
-            margin: 0 auto;
-            padding: 40px 0px;
-            gap: 40px;
+        }
+
+        .heading {
+            text-align: center;
         }
 
         .main-heading {
             font-size: 58px;
             line-height: 80px;
-            max-width: 550px;
-            font-style: normal;
             font-weight: 800;
             letter-spacing: 0.2px;
-            display: flex;
-            justify-content: center;
-            margin-top: 120px;
         }
 
         .sub-heading {
             font-size: 20px;
-            font-style: normal;
             font-weight: 500;
             line-height: 30px;
             letter-spacing: 0.2px;
-            max-width: 530px;
         }
 
-        .button-link {
-            display: inline-block;
-            margin: 10px;
-            padding: 10px 36px;
-            align-items: center;
+        .top-banner__actions {
+            display: flex;
+            gap: 10px;
+            justify-content: center;
+        }
+
+        .button {
+            padding: 10px 35px;
+            line-height: 26px;
+            border-radius: 40px;
+            border: 1px solid white;
             color: white;
-            border-radius: 37px;
-            font-size: 14px;
-            font-weight: 700;
-            text-align: center;
-            text-decoration: none;
+            background: transparent;
             cursor: pointer;
-            line-height: 28px;
             letter-spacing: 0.2px;
+            font-weight: 700;
+            font-family: 'Montserrat';
         }
 
-        .quote-button {
+        .button--primary {
             background-color: #FFA62B;
-            border: none;
+            border: #FFA62B;
         }
 
-        .learn-button {
-            background-color: transparent;
-            border: 2px solid white;
+        @media screen and (min-width: 320px) and (max-width: 570px) {
+            body {
+                background-position: center;
+            }
+
+            .top-banner__header {
+                flex-direction: column;
+            }
+
+            .header__go-home {
+                font-size: 1.5em;
+            }
+
+            .header__nav {
+                margin: 0 auto;
+                font-size: 0.8em;
+                margin: 10px;
+            }
+
+            .header__actions {
+                margin: 0 auto;
+            }
+
+            .main-heading {
+                font-size: 2em;
+                line-height: 40px;
+            }
+
+            .sub-heading {
+                font-size: 1em;
+                line-height: 20px;
+            }
+
+            .button {
+                line-height: 15px;
+            }
```

Comment:
I would also adjust the padding and font-size of the buttons to make them look narrower on smaller screens. Maybe would adjust the headings' size as well. Not sure if it is a necessary thing though,  more likely my personal "design" preference I suppose :)

Reply:
The rest looks good to me!

Reply:
Headings are adjusted with the help of font-size and line-height --->
.main-heading {
                font-size: 2em;
                line-height: 40px;
            }
            .sub-heading {
                font-size: 1em;
                line-height: 20px;
            }
I've decided to do the same thing with the buttons, I set them "line-height: 15px;", but I forgot about font-size. I will definitely change it!
Changing font-size and line-height helped me to make headings and buttons look narrower on smaller screens.

Thank you for your review and "designal" commens <3